app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,34 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :name, class:'form-label' %><br />
+    <%= f.text_field :name, autofocus: true, autocomplete: "text", class:'form-control' %>
+  </div>
+
+  <div class="field">
+    <%= f.label :email, class:'form-label' %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", class:'form-control' %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password, class:'form-label' %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password", class:'form-control' %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation, class:'form-label' %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password", class:'form-control' %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up", class:'btn btn-primary mt-3' %>
+  </div>
+<% end %>
```

Comment:
- [ ] Kindly make sure that a user can sign up successfully, currently, I can not signup successfully  see video below ⬇️ 



https://user-images.githubusercontent.com/69638013/201172657-120d099f-030b-440d-bf89-444b046085b9.mp4